README.md

```diff
@@ -35,11 +28,15 @@
     - [Feature tests](#feature-tests)
     - [Continuous integration](#continuous-integration)
   - [Branching and Releasing](#branching-and-releasing)
-- [Helpful links](#helpful-links)
-- [Contributors](#contributors)
 
 <!-- END doctoc generated TOC please keep comment here to allow auto update -->
 
+## Quick Links
```

Comment:
Just noticed we don't have an "about" statement, describing what this repo/code is for. I suggest adding an `About` section above this one and writing a short description. Or maybe is should go before `Table of contents`?